packages/recommend-js/src/experimental-personalization/frequentlyBoughtTogether.tsx

```diff
@@ -0,0 +1,18 @@
+/** @jsxRuntime classic */
+/** @jsx h */
+
+import { Personalization } from '@algolia/recommend-core';
+
+import {
+  FrequentlyBoughtTogetherProps,
+  frequentlyBoughtTogether as render,
+} from '../frequentlyBoughtTogether';
+import { EnvironmentProps, HTMLTemplate } from '../types';
+
+type Props<TObject> = FrequentlyBoughtTogetherProps<TObject, HTMLTemplate> &
+  EnvironmentProps &
+  Personalization;
+
+export function frequentlyBoughtTogether<TObject>(props: Props<TObject>) {
+  return render<TObject>(props);
+}
```

Comment:
Naming suggestions so that the names are clearer (no brand new names to learn, just add a suffix) and exported with correct naming.

```suggestion
import { Personalization } from '@algolia/recommend-core';

import {
  FrequentlyBoughtTogetherProps as FrequentlyBoughtTogetherPropsPrimitive,
  frequentlyBoughtTogether as frequentlyBoughtTogetherPrimitive,
} from '../frequentlyBoughtTogether';
import { EnvironmentProps, HTMLTemplate } from '../types';

type FrequentlyBoughtTogetherProps<TObject> =
  FrequentlyBoughtTogetherPropsPrimitive<TObject, HTMLTemplate> &
    EnvironmentProps &
    Personalization;

export function frequentlyBoughtTogether<TObject>(
  props: FrequentlyBoughtTogetherProps<TObject>
) {
  return frequentlyBoughtTogetherPrimitive<TObject>(props);
}
```

(Related to my top-level comment, but the code related to `experimental-personalization` should prob also live here.)

Reply:
Thanks ! i addressed this in the last commit 👍